### PR TITLE
Add ubuntu-16.04-qt5.14.2

### PR DESCRIPTION
**Pull request by ubruhin**
Ubuntu 16.04 with the latest Qt version, to be used for creating official LibrePCB binary releases.

### CODE REVIEW

**Comment by rnestler** [comment]
> Ubuntu 16.04 with the latest Qt version, to be used for creating official LibrePCB binary releases.

Official means the AppImage binary?

**Comment by ubruhin** [comment]
> Official means the AppImage binary?

Yes, AppImage + installer + binary *.tar.gz.

File: ubuntu-16.04-qt5.14.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+RUN wget -cq "$LINUXDEPLOYQT_URL" -O /linuxdeployqt.AppImage \
+  && chmod a+x /linuxdeployqt.AppImage \
+  && /linuxdeployqt.AppImage --appimage-extract \
+  && chmod -R 755 /squashfs-root \
+  && ln -s /squashfs-root/AppRun /usr/local/bin/linuxdeployqt \
+  && rm /linuxdeployqt.AppImage
+
+# Install PIP (the version from APT is not sufficient)
+RUN curl "https://bootstrap.pypa.io/get-pip.py" -o get-pip.py \
+  && python get-pip.py
```
Comment (rnestler) [inline ubuntu-16.04-qt5.14.2/Dockerfile:62]: Would this be fixed by using Ubuntu 18.04 or 20.04?

File: ubuntu-16.04-qt5.14.2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:16.04
```
Comment (rnestler) [inline ubuntu-16.04-qt5.14.2/Dockerfile:1]: What is the reason for using 16.04 instead of 18.04 or 20.04?

Comment (ubruhin) [reply ubuntu-16.04-qt5.14.2/Dockerfile:1]: See readme.

File: ubuntu-16.04-qt5.14.2/Dockerfile (hunk 1 above)
Comment (ubruhin) [reply ubuntu-16.04-qt5.14.2/Dockerfile:62]: Yes.

**Pull request merged**